packages/components/src/StatusIndicator/StatusIndicator.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { render } from "@testing-library/react";
+import React from "react";
+import { StatusIndicator } from "./StatusIndicator";
+
+it("renders a success status indicator", () => {
+  const { container } = render(<StatusIndicator status="success" />);
+  expect(container).toMatchSnapshot();
```

Comment:
i wouldn't use a snapshot for this one. I'd actually just assert that. Also, from the looks of it, the snapshot seems wrong too